exporter/stats/prometheus/prometheus.go

```diff
@@ -213,11 +213,17 @@ func (c *collector) toMetric(desc *prometheus.Desc, view *stats.View, row *stats
 		}
 		return hist, nil
 
-	case stats.SumAggregation:
-		panic("stats.SumData not supported yet")
+	case stats.MeanAggregation:
+		data := row.Data.(*stats.MeanData)
+		hist, err := prometheus.NewConstHistogram(desc, uint64(data.Count), data.Sum(), nil, tagValues(row.Tags)...)
```

Comment:
This should be a gauge value, isn't it? We do not have the distribution buckets to create a useful histogram metric type.

Reply:
My bad, I wonder why I even thought so, fixing this in a second.